Doc/library/logging.config.rst

```diff
@@ -524,6 +524,10 @@ valid keyword parameter name, and so will not clash with the names of
 the keyword arguments used in the call.  The ``'()'`` also serves as a
 mnemonic that the corresponding value is a callable.
 
+    .. versionadded:: 3.11
```

Comment:
> I've added it to the section of user custom objects, have a look and let me know if you think it fits there.
Should we also add it to both the handlers and loggers section?

Yes, it does belong here, but on reflection I think it should be a `versionchanged` rather than `versionadded` and this should also be in the bullets of `handlers` and `loggers` in the "Dictionary Schema Details" section where it currently says that `filters` are "A list of ids of the filters for this handler".